src/main/java/be/xplore/pricescraper/utils/ProductMatcher.java

```diff
@@ -0,0 +1,74 @@
+package be.xplore.pricescraper.utils;
+
+import org.apache.commons.text.similarity.LevenshteinDistance;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
I would not mark this class as a Component, since it contains hard business logic. For this specific class, it would be more useful to instantiate with its constructor, this allows for more fine-grained control of its parameters per use case.